### ISSUE TITLE
Update versions various tools and dependencies

### ISSUE THREAD

**Issue by tiembo** [googlecodelabs/watchface#3]
Items to address:
* SDK and built tools currently use API 25
* Gradle
* Google Maven

**Comment by ELI7VH** [comment]
This does seem very out of date, would be nice to have something that fits recent changes

**Comment by lancelinksp** [comment]
Yes please update this especially for recent changes such as the deprecation of 'new Handler()'

**Comment by chartsNgraphs** [comment]
Can we get any traction on this? Would really appreciate it as much is out of date.
